linker/shared/src/main/scala/org/scalajs/linker/standard/LinkTimeProperties.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.linker.standard
+
+import org.scalajs.ir.{Types => jstpe, Trees => js}
+import org.scalajs.ir.Trees.LinkTimeProperty._
+import org.scalajs.ir.ScalaJSVersions
+import org.scalajs.ir.Position.NoPosition
+import org.scalajs.linker.interface.{Semantics, ESFeatures}
+
+final class LinkTimeProperties private[standard] (
```

Comment:
If we keep this in standard, we should be super restrictive in its interface (e.g. do not expose that it is a class, etc.).

Reply:
TBH, I'm not sure where should I put `LinkTimeProperties` (we can move this to `linker-interface` but anyway need to restrictive in its interface?)

Reply:
Do we need to make it public at all? Can we keep it `private[linker]` instead?

Reply:
ah, that's right, made it `private[linker]` :+1: